samples/sensor/mpu6050/boards/nrf52_pca10040.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c0 {
+	mpu6050@68 {
+		compatible = "invensense,mpu6050";
+		reg = <0x68>;
+		status = "okay";
+		label = "MPU6050";
+		int-gpios = <&gpio0 11 0>;
```

Comment:
missing flags

Reply:
Actually not missing here: this is targeting master, and `GPIO_ACTIVE_HIGH` isn't available yet.

Reply:
Right, thanks.